packages/core/src/Stores/client-store.js

```diff
@@ -446,6 +445,15 @@ export default class ClientStore extends BaseStore {
             }
         );
 
+        reaction(
+            () => [this.is_logged_in, this.is_passkey_supported],
+            () => {
+                if (this.is_logged_in && this.is_passkey_supported) {
+                    this.fetchShouldShowEffortlessLoginModal();
+                }
+            }
+        );
+
```

Comment:
Can this be a hook instead?

Reply:
I think we will think about such kind of refactoring while moving the code in v2 - a lot of things should be changed